cluster/manifests/deletions.yaml

```diff
@@ -1,7 +1,7 @@
 # everything defined under here will be deleted before applying the manifests
 pre_apply:
 {{ if ne .Cluster.ConfigItems.routegroups_validation "enabled" }}
-- name: "routegroup-admitter.teapot.zalan.do"
+- name: "skipper-admitter.teapot.zalan.do"
```

Comment:
I think you'd need to have temporary unconditional deletion for `routegroup-admitter.teapot.zalan.do` due to rename.

Reply:
Good catch

Reply:
We will need also not to remove `skipper-admitter.teapot.zalan.do` in the first run since it's not there in the first place